py/core/main/app_entry.py

```diff
@@ -73,7 +73,6 @@ def r2r_app(
 app = r2r_app(
     config_name=config_name,
     config_path=config_path,
-    client_mode=client_mode,
```

Comment:
If we're removing client_mode here, we should be removing it elsewhere in the file as well. I.e. line 61 and 69